### ISSUE TITLE
Instance up fails in MacOS

### ISSUE THREAD

**Issue by werty37** [contember/contember#132]
MacOS Mojave 10.14.6
Node Version: v14.15.0
NPM Vesion: 6.14.8

```
npx contember instance:up
$ docker-compose 'stop' 'api'
$ docker-compose 'rm' '-f' 'api'
No stopped containers
$ docker-compose 'up' '-d'
Creating network "default_default" with the default driver
Creating volume "default_pgsql-data" with default driver
Creating volume "default_s3-data" with default driver
Pulling db (postgres:10-alpine)...
10-alpine: Pulling from library/postgres
Digest: sha256:e5f5a0147a23143e0371a1f4bddf19e7dd6d42e800c9bd6b1bb9b7e86d544c28
Status: Downloaded newer image for postgres:10-alpine
Pulling mailhog (mailhog/mailhog:)...
latest: Pulling from mailhog/mailhog
Digest: sha256:8d76a3d4ffa32a3661311944007a415332c4bb855657f4f6c57996405c009bea
Status: Downloaded newer image for mailhog/mailhog:latest
Pulling api (contember/contember:0.9.0-alpha.1)...
manifest for contember/contember:0.9.0-alpha.1 not found: manifest unknown: manifest unknown
ChildProcessError: Command has failed(1): Creating network "default_default" with the default driver
Creating volume "default_pgsql-data" with default driver
Creating volume "default_s3-data" with default driver
Pulling db (postgres:10-alpine)...
Pulling mailhog (mailhog/mailhog:)...
Pulling api (contember/contember:0.9.0-alpha.1)...
manifest for contember/contember:0.9.0-alpha.1 not found: manifest unknown: manifest unknown
 
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/@contember/cli/src/utils/commands.ts:52:12)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12) {
  exitCode: 1,
  stderr: 'Creating network "default_default" with the default driver\n' +
    'Creating volume "default_pgsql-data" with default driver\n' +
    'Creating volume "default_s3-data" with default driver\n' +
    'Pulling db (postgres:10-alpine)...\n' +
    'Pulling mailhog (mailhog/mailhog:)...\n' +
    'Pulling api (contember/contember:0.9.0-alpha.1)...\n' +
    'manifest for contember/contember:0.9.0-alpha.1 not found: manifest unknown: manifest unknown\n'
}
```
Am I missing something here?

**Comment by JanTvrdik** [comment]
The tag name should **v**0.9.0-alpha.1

**Comment by werty37** [comment]
How do I change from 0.9.0-alpha.1 to v0.9.0-alpha.1? I am following the getting started doc.

[Contember Getting Started](https://www.contember.com/docs/intro/getting-started)

**Comment by Cactucs** [comment]
You can try changing `image: contember/contember:0.9.0-alpha.1` to `image: contember/contember:v0.9.0-alpha.1` in `instances/default/docker-compose.yaml` file.

**Comment by werty37** [comment]
@Cactucs Thank you. This resolved the `manifest for contember/contember:0.9.0-alpha.1 not found: manifest unknown: manifest unknown\n` issue on mac.
